app/api/controllers/validators/resolvers/Resolvers.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package api.controllers.validators.resolvers
+
+import api.models.errors.MtdError
+import cats.data.Validated
+import cats.data.Validated.Valid
+import cats.implicits._
+
+import scala.math.Ordered.orderingToOrdered
+
+trait Resolvers {
```

Comment:
As every resolver extends this, could it be renamed Resolver? Seems a bit odd for it to be plural, "I am a resolvers".

Reply:
Agree it should probably be renamed. It's not necessary for all resolvers so not sure Resolver is right either. Was thinking ResolverSupport??

Reply:
Then I can also rename SimpleResolver type alias to Resolver.

Reply:
sounds good